ui/cryptomaterial/dropdown.go

```diff
@@ -228,20 +254,55 @@ func (d *DropDown) Layout(gtx C, dropPos int, reversePos bool) D {
 		iRight = dropPos
 	}
 
+	if d.OpenedLayoutInset == nil {
+		d.OpenedLayoutInset = &layout.Inset{
+			Left:  unit.Dp(float32(iLeft)),
+			Right: unit.Dp(float32(iRight)),
+		}
+	}
+
+	if d.Stack {
+		isOpen := d.isOpen
+		return layout.Stack{Alignment: alig}.Layout(gtx,
+			layout.Expanded(func(gtx layout.Context) layout.Dimensions {
```

Comment:
```suggestion
			layout.Expanded(func(gtx C) D {
```

please help us change all layout.Context -> C and layout.Dimensions -> D{}. It will be more consistent in this file